src/ape/managers/accounts.py

```diff
@@ -317,3 +341,20 @@ def __contains__(self, address: AddressType) -> bool:
             any(address in container for container in self.containers.values())
             or address in self.test_accounts
         )
+
+    def use_sender(
+        self,
+        account_id: Union[AccountAPI, AddressType, str, int],
+        autosign: bool,
+        passphrase: Optional[str] = None,
```

Comment:
This might be a little devious, but hear me out:
```suggestion
       **kwargs,
```